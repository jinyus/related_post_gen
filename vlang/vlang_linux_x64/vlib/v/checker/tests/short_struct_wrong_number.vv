struct Test {
	foo bool
	bar bool
}

fn main() {
	_ = Test{true}
	_ = Test{true, false, true}
}
