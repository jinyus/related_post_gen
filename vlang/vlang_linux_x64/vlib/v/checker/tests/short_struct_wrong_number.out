vlib/v/checker/tests/short_struct_wrong_number.vv:7:6: error: too few fields in `Test` literal (expecting 2, got 1)
    5 |
    6 | fn main() {
    7 |     _ = Test{true}
      |         ~~~~~~~~~~
    8 |     _ = Test{true, false, true}
    9 | }
vlib/v/checker/tests/short_struct_wrong_number.vv:8:6: error: too many fields in `Test` literal (expecting 2, got 3)
    6 | fn main() {
    7 |     _ = Test{true}
    8 |     _ = Test{true, false, true}
      |         ~~~~~~~~~~~~~~~~~~~~~~~
    9 | }
